src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/RefreshTokenNotification.cs

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+    class RefreshTokenParams 
+    {
+        /// <summary>
+        /// Tenant for the token
```

Comment:
The tenant isn't "for the token" - it's just a part of the scope of what the token is authenticating for. Just `ID of the tenant` is fine - that's just a standard part of any AD auth flow so shouldn't need a detailed explanation.